### ISSUE TITLE
tonemapping: use full 768*512 pic for all examples?

### ISSUE THREAD

**Issue by Artoria2e5** [64/64.github.io#1]
The [tonemapping article](https://64.github.io/tonemapping) currently uses square-cropped 180x180 pics for some examples but a full 768x512 pic for others. This is quite nice when put in text, but seems to mess the grid in the conclusion up. It might be a good idea to upload all pics as 768x512 and do a little css to crop them down to squares in text.

(And heck yeah it is good!)

**Comment by 64** [comment]
Hmm, I don't think any of the pictures are square. Can you show an example of what you mean?

**Comment by Artoria2e5** [comment]
On my end it looks like this:
<img width="797" alt="Screen Shot 2021-08-18 at 16 03 00" src="https://user-images.githubusercontent.com/6459309/129861433-12fb753f-f94a-45e2-84be-175f96ff8b3c.png">

**Comment by 64** [comment]
That is pretty broken indeed. What browser are you using?

**Comment by Artoria2e5** [comment]
Happens on Safari 14, Windows Chrome, Windows Firefox, and Mac Chrome 92. Can it be working for you only due to caching?

**Comment by 64** [comment]
I don't think so, it also works just fine on 3-4 different computers and multiple browsers I've tried. I don't think it is a cache problem.

Are there any messages in the developer console for you? Does it also happen on your phone? Maybe it is a network / CDN / firewall issue or something, how bizarre.

**Comment by Artoria2e5** [comment]
Weird, it doesn’t happen on my iPhone! Gonna F12 it…

**Comment by Artoria2e5** [comment]
Nothing in the console. The  "pattern" I am seeing is that the 180x180 pics are coming from `ibb.co` while the full-sized one are coming from `i.imgur.com`; this also shows in `view-source:https://64.github.io/tonemapping/`, so I am going to assume JS is not involved.

The `ibb.co` links are quite... peculiar. If I visit `https://i.ibb.co/S6JdvQz/out.png`, it redirects me to a webpage that frames the image. If I paste the same link into the address bar again, it gives the full-sized pic...

**Comment by 64** [comment]
Hm, that is odd. It'll probably be fixed if I host the images locally instead; maybe I should just do that. I don't really trust that image hosting site to keep the images indefinitely or remember why I used it in the first place.
